deploy/charts/values.yaml

```diff
@@ -94,15 +94,35 @@ kube-prometheus-stack:
 
     config:
       global:
+        ## smtp host which would relay emails to the recepient eg: smtp.office.com:587.
+        smtp_smarthost: ''
+        ## from address on the email notification.
+        smtp_from: ''
+        ## username to authenticate with smtp host
+        smtp_auth_username: ''
+        ## pawwrod for the username.
+        smtp_auth_password: ''
         resolve_timeout: 5m
       route:
         group_by: ["alertname", "job", "volName"]
         group_wait: 30s
         group_interval: 5m
         repeat_interval: 4h
-        receiver: "null"
+        receiver: team-X-mails
+
+        routes:
+          - matchers:
+              - service="openebs"
+            receiver: openebs-email
       receivers:
-        - name: "null"
+        - name: 'team-X-mails'
+          email_configs:
+            - to: ''
```

Comment:
What's the behaviour when these are empty, does helm install fail?
What about upgrades?

Reply:
We cant have them empty. Helm install/upgrade goes through. However, alertmanager doesnt get Ready. It complains if it doesnt find `to`, `smtp_smarthost` and `smtp_from` if have `email_config` in receivers. `smtp_auth_username` and `smtp_auth_password` are not mandatory so that we may keep blank.